modules/mobile/spec/request/prescriptions_request_spec.rb

```diff
@@ -22,23 +22,29 @@
 
   before(:all) do
     @original_cassette_dir = VCR.configure(&:cassette_library_dir)
-    VCR.configure { |c| c.cassette_library_dir = 'modules/mobile/spec/support/vcr_cassettes' }
+    VCR.configure do |c|
+      c.cassette_library_dir = 'modules/mobile/spec/support/vcr_cassettes'
+      c.register_request_matcher(:user_agent_header) do |request1, request2|
```

Comment:
I'm curious how much scope this has. We have to reset the library_dir in an after block. So I assume this lingers in the vcr config. Which should be harmless. But the fact that a change we're making here could linger for the rest of the test suite doesn't feel right. It might be worth extracting this to a spec_helper or rails_helper file somewhere. I don't believe our module has a spec or rails helper and when I tried to add one I ran into some issues (that I'm sure we could overcome). But... given that this change would likely be global it might be best to do them at the global level anyhow.